lang/cs/Org.Apache.REEF.Bridge/Org.Apache.REEF.Bridge.vcxproj

```diff
@@ -41,6 +41,7 @@ under the License.
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <Keyword>ManagedCProj</Keyword>
     <RootNamespace>Org.Apache.REEF.Bridge</RootNamespace>
+    <WindowsTargetPlatformVersion Condition="'$(VisualStudioVersion)' >= '15.0'">10.0.14393.0</WindowsTargetPlatformVersion>
```

Comment:
This means that if the build is done on VS 2017, it targets Windows 10, right? Does that also mean those builds need Windows 10 / Windows Server 2016 to execute? If so, we might want to add that to our README and file an issue against the README to make sure we fix it once we removed the bridge.